oceannavigator/frontend/src/components/AreaWindow.jsx

```diff
@@ -291,13 +243,13 @@ class AreaWindow extends React.Component {
       <Panel.Heading>{_("Area Settings")}</Panel.Heading>
       <Panel.Collapse>
         <Panel.Body>
-          <Row>   {/* Contains compare dataset and help button */}
+          <Row>
             <Col xs={9}> 
               <SelectBox
```

Comment:
It looks like this should be for the "Compare Datasets" selection box.  It is working for me when I defined an area by selecting a region with the mouse. But I use on of the predefined region (e.g. AZMP NL region NP46) clicking on "Compare Datasets" leads to blank page. (same behaviour occurs before this PR so not a blocking issue)